marble-table/src/test/java/org/apache/calcite/table/TypeConvertUtilTest.java

```diff
@@ -0,0 +1,158 @@
+package org.apache.calcite.table;
+
+import org.apache.calcite.util.NlsString;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.math.BigDecimal;
+import java.sql.Date;
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TypeConvertUtilTest {
+
+  @Test
+  public void testCastBigDecimal() {
```

Comment:
rearrange the order so the 1st test method is for the 1st method

Reply:
I agree this would be better but don't think it is critical, especially with a relatively large number of test methods.
I note that Diffblue Cover seems to place them in alphabetical order.